tests/test_xlora.py

```diff
@@ -144,7 +144,7 @@ def test_scalings_logging_methods(self, tokenizer, model):
 
         model.disable_scalings_logging()
 
-        inputs = tokenizer.encode("Python is a", add_special_tokens=False, return_tensors="pt")
+        inputs = tokenizer.encode("Python is a", add_special_tokens=False, return_tensors="pt", clean_up_tokenization_spaces=True)
```

Comment:
In transformers 4.45, the default is False. Perhaps making it `True` fixes it, as the FutureWarning in [the log](https://github.com/huggingface/peft/actions/runs/10557574065/job/29245435767) indicates?